utils/documentation_tests.txt

```diff
@@ -39,6 +39,7 @@ src/transformers/models/ctrl/configuration_ctrl.py
 src/transformers/models/ctrl/modeling_ctrl.py
 src/transformers/models/cvt/modeling_cvt.py
 src/transformers/models/data2vec/modeling_data2vec_audio.py
+src/transformers/models/data2vec/configuration_data2vec_text.py
```

Comment:
This should go before the line `src/transformers/models/data2vec/modeling_data2vec_audio.py`